docs/releasehistory.rst

```diff
@@ -15,6 +15,9 @@ New features
   control global variables (`#363 <https://github.com/choderalab/openmmtools/pull/363>`_).
 - Allow restraint force classes to be controlled by a parameter other than ``lambda_restraints``. This will enable
   multi-restraints simulations (`#363 <https://github.com/choderalab/openmmtools/pull/363>`_).
+- ``RadiallySymmetricRestraintForce`` and all subclasses are now a ``CustomCVForce`` subclass wrapping their
```

Comment:
We should warn that this may be an API-breaking change in the changelog!

Reply:
I did?

``RadiallySymmetricRestraintForce`` and all subclasses are now a ``CustomCVForce`` subclass wrapping their  respective ``CustomBondForce`` and ``CustomCentroidBondForce`` objects. **This breaks backwards compatibility**, but enables tracking the restraint distances through the new ``SamplerSate`` collective variable features.

Emphasis mine

Reply:
Just meant that there might be a "Warnings" section rather than burying this in the "Features" (since it may break your program, which may not be a "feature").

Reply:
Fair point. I'll make this more visible